myapp/models.py

```diff
@@ -3,16 +3,16 @@
 from django.urls import reverse
 
 
-
-#base model
+# base model
 class BaseModel(models.Model):
     created_at = models.DateTimeField(default=django.utils.timezone.now())
     updated_at = models.DateTimeField(default=django.utils.timezone.now())
-    
+
     class Meta:
-        abstract = True 
-        
-#Author details model
+        abstract = True
+
+
+# Author details model
 class Author(models.Model):
```

Comment:
Please inherits the `BaseModel`. You have to inherit the `BaseModel` in each and every model.